presto-hive/src/main/java/com/facebook/presto/hive/orc/DwrfBatchPageSourceFactory.java

```diff
@@ -70,12 +70,13 @@ public DwrfBatchPageSourceFactory(TypeManager typeManager, HiveClientConfig conf
             long start,
             long length,
             long fileSize,
-            Properties schema,
+            Storage storage,
+            Map<String, String> tableParameters,
             List<HiveColumnHandle> columns,
             TupleDomain<HiveColumnHandle> effectivePredicate,
             DateTimeZone hiveStorageTimeZone)
     {
-        if (!isDeserializerClass(schema, OrcSerde.class)) {
+        if (!OrcSerde.class.getName().equals(storage.getStorageFormat().getSerDe())) {
```

Comment:
nit: is this just an inline? -- do we need this change?

Reply:
The `Properties schema` is no longer in the parameters. Now there's `Storage` instead.